src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -712,35 +711,21 @@ private void CheckStake(Block block, ChainedHeader chainTip)
                 return false;
             }
 
-            // Split stake if above threshold.
-            bool splitStake = this.GetSplitStake(nonEmptyUtxos, chainTip);
-            if (splitStake)
-            {
-                this.logger.LogTrace("Coinstake UTXO will be split to two.");
-                coinstakeContext.CoinstakeTx.Outputs.Add(new TxOut(0, coinstakeContext.CoinstakeTx.Outputs[1].ScriptPubKey));
-            }
-
             // Input to coinstake transaction.
             UtxoStakeDescription coinstakeInput = workersResult.KernelCoin;
 
             // Total amount of input values in coinstake transaction.
-            long coinstakeInputValue = coinstakeInput.TxOut.Value + reward;
+            long coinstakeInputValue = workersResult.KernelCoin.TxOut.Value + reward;
 
-            // Set output amount.
-            if (coinstakeContext.CoinstakeTx.Outputs.Count == 3)
-            {
-                coinstakeContext.CoinstakeTx.Outputs[1].Value = (coinstakeInputValue / 2 / Money.CENT) * Money.CENT;
-                coinstakeContext.CoinstakeTx.Outputs[2].Value = coinstakeInputValue - coinstakeContext.CoinstakeTx.Outputs[1].Value;
-                this.logger.LogTrace("Coinstake first output value is {0}, second is {1}.", coinstakeContext.CoinstakeTx.Outputs[1].Value, coinstakeContext.CoinstakeTx.Outputs[2].Value);
-            }
-            else
-            {
-                coinstakeContext.CoinstakeTx.Outputs[1].Value = coinstakeInputValue;
-                this.logger.LogTrace("Coinstake output value is {0}.", coinstakeContext.CoinstakeTx.Outputs[1].Value);
-            }
+            Transaction coinstakeTx = this.PrepareCoinStakeTransactions(
```

Comment:
We use different style for calling, so for the consistency reasons, try to match the style used elsewhere